week3/aniketsaoji/anagrams.java

```diff
@@ -0,0 +1,44 @@
+package anagrams;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class anagrams {
```

Comment:
Have you considered using UpperCamelCase for [class names](https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names)?